### ISSUE TITLE
UU: Fokusrektangel har forsvunnet på del skjerm med veileder

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-dekoratoren#921]
Gjelder ved forenklet (simple) dekoratør.

https://www.nav.no/start/ettersend-soknad-sykepenger

https://my2.siteimprove.com/Inspector/1002489/A11Y/Page?pageId=68437166800&impmd=6DD5538FAB0E174F64D81B4146785183&expand=potential&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r65/cantTell/NE9URzN3ZVI=;MDlmMmViN2U=
